lib/datadog/appsec/assets/waf_rules/recommended.json

```diff
@@ -1,7080 +1,4953 @@
-              {
-                "address": "server.request.headers.no_cookies",
-                "key_path": [
-                  "user-agent"
-                ]
-              }
-            ],
-            "regex": "^wpscan\\b"
-          },
-          "operator": "match_regex"
-        }
-      ],
-      "transformers": []
-    },
-    {
-      "id": "ua0-600-62x",
-      "name": "Aon pentesting services",
-      "tags": {
-        "type": "commercial_scanner",
-        "category": "attack_attempt",
-        "tool_name": "Aon",
-        "confidence": "0"
-      },
-      "conditions": [
-        {
-          "parameters": {
-            "inputs": [
-              {
-                "address": "server.request.headers.no_cookies",
-                "key_path": [
-                  "user-agent"
-                ]
-              }
-            ],
-            "regex": "^Aon/"
-          },
-          "operator": "match_regex"
-        }
-      ],
-      "transformers": []
-    },
-    {
-      "id": "ua0-600-6xx",
-      "name": "Stealthy scanner",
-      "tags": {
-        "type": "security_scanner",
-        "category": "attack_attempt",
-        "confidence": "1"
-      },
-      "conditions": [
-        {
-          "parameters": {
-            "inputs": [
-              {
-                "address": "server.request.headers.no_cookies",
-                "key_path": [
-                  "user-agent"
-                ]
-              }
-            ],
-            "regex": "mozilla/4\\.0 \\(compatible(; msie 6\\.0; win32)?\\)"
-          },
-          "operator": "match_regex"
-        }
-      ],
-      "transformers": []
-    },
-    {
-      "id": "ua0-600-7xx",
-      "name": "SQLmap",
-      "tags": {
-        "type": "security_scanner",
-        "category": "attack_attempt",
-        "tool_name": "SQLmap",
-        "confidence": "1"
-      },
-      "conditions": [
-        {
-          "parameters": {
-            "inputs": [
-              {
-                "address": "server.request.headers.no_cookies",
-                "key_path": [
-                  "user-agent"
-                ]
-              }
-            ],
-            "regex": "^sqlmap/"
-          },
-          "operator": "match_regex"
-        }
-      ],
-      "transformers": []
-    },
-    {
-      "id": "ua0-600-9xx",
-      "name": "Skipfish",
-      "tags": {
-        "type": "security_scanner",
-        "category": "attack_attempt",
-        "tool_name": "Skipfish",
-        "confidence": "1"
-      },
-      "conditions": [
-        {
-          "parameters": {
-            "inputs": [
-              {
-                "address": "server.request.headers.no_cookies",
-                "key_path": [
-                  "user-agent"
-                ]
-              }
-            ],
-            "regex": "(?i)mozilla/5\\.0 sf/"
-          },
-          "operator": "match_regex"
-        }
-      ],
-      "transformers": []
-    }
-  ]
-}
+version: '2.2'
```

Comment:
This looks like YAML. We currently support JSON 😄

Reply:
Brain fart, sorry.